src/main/java/io/javalin/Javalin.java

```diff
@@ -165,6 +167,16 @@ public Javalin stop() {
         return this;
     }
 
+    /**
+     * Configure instance to not show banner in logs.
+     * The method must be called before {@link Javalin#start()}.
+     */
+    public Javalin dontShowBannerInLogs() {
```

Comment:
Everything else is called `enableXyz`, so `disableStartupBanner` perhaps?

Reply:
why not!